test/utils/request_custom.js

```diff
@@ -0,0 +1,32 @@
+var Request = require('request');
+
+/**
+ * Comment out the parts of request.js that automatically set a form-urlencoded
+ * request to be `charset=utf-8` in this overriden defintion.
+ * Refer to https://github.com/request/request/commit/86571c794b75bb637b39eb7574e825e461647151
+ */
```

Comment:
I stopped using this file and will remove it from the diff if it's definitely not needed.